README.md

```diff
@@ -21,13 +21,21 @@ To install the plugin, run:
 
 ### Commands
 
-- `ceph`: Run a Ceph CLI command. Supports any arguments the `ceph` command supports. See [Ceph docs](https://docs.ceph.com/en/pacific/start/intro/) for more.
+- `ceph <args>` : Run a Ceph CLI command. Supports any arguments the `ceph` command supports. See [Ceph docs](https://docs.ceph.com/en/pacific/start/intro/) for more.
 
-- `operator`: run sub-commands against the Rook-Ceph operator
-  -  `restart`: restart the Rook-Ceph operator
+- `rbd <args>` : Call a 'rbd' CLI command with arbitrary args"
 
-- `help`: Output help text
+- `operator <subcommands>` : run sub-commands against the Rook-Ceph operator mentioned below.
+  - `restart` : restart the Rook-Ceph operator.
+  - `set <property> <value>` : Set the property in the rook-ceph-operator-config configmap.
 
+- `rook <subcommands>` : run sub-commands mentioned below.
+  - `version` : print the version of Rook.
+  - `status` : print the phase of CR in the namespace. Default is CephCluster CR status.
+
+`get-mon-endpoints` : print mon endpoints.
+
+- `help` : Output help text.
 
 ## Examples
```

Comment:
How about adding an example for printing versions? It's one of the simplest and most common questions.
```
### Versions

kubectl rook-ceph rook version

<output>

kubectl rook-ceph ceph versions

<output>